caffeine/build.gradle

```diff
@@ -117,6 +117,11 @@ task generateNodes(type: JavaExec) {
 }
 compileJava.dependsOn(generateNodes)
 
+afterEvaluate {
```

Comment:
Is `afterEvaluate` really needed here?
AFAIK it should work without that as well.

Reply:
The `sourceJar` task isn't visible because the `publish.gradle` is applied using at the root using `subprojects`. That is applied after the project's build script is evaluated, causing this ordering issue.